constants.js

```diff
@@ -6,15 +6,18 @@ const NETWORKS = {
 
 const ERROR_MESSAGES = {
     INVALID_POWPEG_PUBLIC_KEYS: 'powpegBtcPublicKeys should be an array',
-    INVALID_ERP_PUBLIC_KEYS: 'erpBtcPublicKeys should be an array',
-    INVALID_CSV_VALUE: 'csvValue is required',
+    INVALID_P2SH_ERP_PUBLIC_KEYS: 'p2shErpBtcPublicKeys should be an array',
```

Comment:
```suggestion
    INVALID_EMERGENCY_PUBLIC_KEYS: 'emergencyBtcPublicKeys should be an array',
```
Let's keep this field name consistent